### ISSUE TITLE
query-overpass using in compatible xhr?

### ISSUE THREAD

**Issue by calocan** [perliedman/query-overpass#25]
It seems like the latest version of query-overpass uses an xhr module that has a different signature than the standard request module. This doesn't show up in the query-overpass tests since it doesn't acknowledge the package.json configuration:
```
"browser": {
    "request": "xhr"
},
```
in the tests. But in the browser xhr is used instead of request and fails because xhr is expecting a callback:
```
if(typeof options.callback === "undefined"){
    throw new Error("callback argument missing")
}
```
xhr/index.js line 55

query_overpass/index.js line 33 passes reqOptions without a callback, since it expects to simply wait for the response.

```
index.js:55 Uncaught Error: callback argument missing
    at _createXHR (http://localhost:3000/static/index.js:142365:15)
    at Function.createXHR.(anonymous function) [as post] (http://localhost:3000/static/index.js:142325:16)
    at module.exports (http://localhost:3000/static/index.js:98970:21)
    at http://localhost:3000/static/index.js:66261:48

```

**Comment by perliedman** [comment]
Hm, interesting. Since we moved over to the current stream based approach, I guess that using `xhr` will not work. 

I guess some work would have to be done to support streams when run server side, but switchable to `xhr` and without streams for more lightweight scenarios in the browser.

I don't think I will have much time to address this right now, so any help would be appreciated.

**Comment by planemad** [comment]
Is there a workaround for this? Have tries to use older versions of query-overpass, but it gives the same error.

**Comment by perliedman** [comment]
@planemad version 1.3.0 or earlier should be fine, from looking at the commits.

**Comment by perliedman** [comment]
Had some extra time on a train ride today, so this should be fixed in version 1.5.1 that I published just now!

**Comment by planemad** [comment]
❤️ amazing @perliedman
